### PR TITLE
Improve theme color experience

### PR DESCRIPTION
**Pull request by decepulis**
1. explain theme colors with new `ThemeColorPopover` component
2. carry theme colors from home to player page with new `ThemeLink` component
3. add theme colors to the code snippet with changes to `DocsEmbed`

### CODE REVIEW

File: site/app/_components/DocsEmbed.tsx [hunk 1]
```diff
@@ -148,7 +168,7 @@ function reactCode(theme: string, mediaElement: any, mediaPackage: string) {
   ];
 }
 
-function vueCode(theme: string, mediaElement: any, mediaPackage: string) {
+function vueCode(theme: string, mediaElement: any, mediaPackage: string, customProperties: CustomProperties) {
```
Comment (decepulis) [inline site/app/_components/DocsEmbed.tsx:171]: Vue's the only language I don't know for sure, but it worked in CodeSandbox, and Copilot told me it wasn't not idiomatic.

File: site/app/_components/ThemeColorPopover.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+              </code>
+            </li>
+          </ul>
+          <p className="mb-0.5">
+            You can read more about customizing themes by visiting the{' '}
+            <Link href="https://www.media-chrome.org/docs/en/reference/styling" className="underline">
+              Media Chrome Styling Reference
+            </Link>
+            .
+          </p>
```
Comment (decepulis) [inline site/app/_components/ThemeColorPopover.tsx:73]: Very open to workshopping this text

Comment (heff) [reply site/app/_components/ThemeColorPopover.tsx:73]: Looks great 👍

**Comment by heff** [comment]
Awesome!

**Review by heff — APPROVED** [review]

**Pull request merged**